app-misc/fff/fff-1.0.recipe

```diff
@@ -0,0 +1,35 @@
+SUMMARY="A simple file manager written in bash"
+DESCRIPTION="Features:
+* Fast
+* Minimal (only requires bash and coreutils)
+* Smooth Scrolling (using vim keybindings)
+* Supports LS_COLORS
+* File Oprations (copy, paste, rename, cut, etc)
+* Instant as you type search"
+HOMEPAGE="https://github.com/dylanaraps/fff"
+COPYRIGHT="2016-2018 Dylan Araps"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="$HOMEPAGE/archive/$portVersion.tar.gz"
+CHECKSUM_SHA256="6e5eed18b50b3e1a2497092e618665ff00b71b27b4f7653f9693ee40517d1a8a"
+SOURCE_FILENAME="fff-$portVersion.tar.gz"
+
+ARCHITECTURES="any"
+
+PROVIDES="
+	fff = $portVersion
+	cmd:fff
```

Comment:
```suggestion
	cmd:fff = $portVersion
```

Reply:
Should this be done for all `cmd:` lines?

Reply:
Yep, it's a good practice to version everything.

Reply:
I don't see much use for that, tbh.
It would be relevant if the command line options were used in scripts and they were renamed a lot, so there is a need to distinguish different versions of a command (eg. iperf 2 vs iperf 3). But in other cases, I think it's a waste of effort to specify versionning.